hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerRatisServer.java

```diff
@@ -671,6 +672,21 @@ public RaftServerStatus checkLeaderStatus() {
     return RaftServerStatus.NOT_LEADER;
   }
 
+  @VisibleForTesting
+  public List<String> getCurrentPeersFromRaftConf() {
+    try {
+      Collection<RaftPeer> currentPeers =
+          server.getDivision(raftGroupId).getRaftConf().getCurrentPeers();
+      List<String> currentPeerList = new ArrayList<>();
+      currentPeers.forEach(e -> currentPeerList.add(e.getId().toString()));
+      return currentPeerList;
+    } catch (IOException e) {
+      // In this case we return not a leader.
+      LOG.error("Failed to get RaftServer information. ", e);
```

Comment:
Question: Why in exception case logging and return null, can't we throw an exception?
And also can we add JavaDoc for this?

Reply:
Done.